src/main/kotlin/com/vk/admstorm/diagnostic/SentryErrorReporter.kt

```diff
@@ -41,10 +41,12 @@ class SentryErrorReporter : ErrorReportSubmitter() {
                 continue
             }
 
-            val sentryId = sentry.sendError(event.message, event.data.throwable)
+            val sentryId = event.message?.let { sentry.sendError(it, event.data.throwable) }
```

Comment:
let's better specify that `message` in `sendEvent` can be `null` of `SentryService` than this way.